journey-map/public/style.css

```diff
@@ -189,3 +189,11 @@ form fieldset p {
     stroke-width: 3px;
     filter: drop-shadow(1px 0 0 #ffdd00) drop-shadow(-1px 0 0 #ffdd00) drop-shadow(0 1px 0 #ffdd00) drop-shadow(0 -1px 0 #ffdd00);
 }
+#header-actions {
+    position: fixed;
+    right: 30px;
+}
+#journey-select {
```

Comment:
I think this (and the search box) could have a slightly larger font size, e.g. `1rem`?

Reply:
Yes looks better